Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -386,6 +386,14 @@ public struct MissingAttributeArgument: ParserError {
   }
 }
 
+public struct MissingBothStringQuotesOfStringSegments: ParserError {
+  public let stringSegments: StringLiteralSegmentsSyntax
+
+  public var message: String {
+    return "expected \(nodesDescription([stringSegments], format: false)) to be surrounded by '\"'"
```

Comment:
If you just want to print the source test of `stringSegments` verbatim, I would use `. shortSingleLineContentDescription ` instead. The core functionality of `nodesDescription` is to paraphrase entire nodes by their node name but I don’t think that’s something we want here.

```suggestion
    return "expected '\(stringSegments.shortSingleLineContentDescription)' to be surrounded by '\"'"
```

Reply:
Oh, I agree. Using shortSingleLineContentDescription instead of nodesDescription seems to be in line with the intention we want.